EVES/drafts/EVES-005/eves-005.md

```diff
@@ -76,10 +76,26 @@ The consumer uses metadata search or similar services to identify an asset of in
 
 At this point, the contract is fully settled and the consumer can use the asset.
 
-### 5. Fee Payment
+### 5. Fee and Payment Report
 
-On a regular basis, the providers send the accumulated fees to the operator.
-To ensure the fees are correctly calculated without publishing business statistics, a provider should construct a zero knowledge proof for the accumulated amount being correct based on the submitted hashes.
+#### 5.1. Asset Fee
+
+1. After a defined period, the provider compiles all completed contracts and generates a cumulative bill.
```

Comment:
What does "completed contracts" mean? Just signed contracts? Or service usage billed according to a service contract? This should probably be more general wording. Contracts might also define specific payment schedules.